scenarios/15_Ssyuriss_Fen.cfg

```diff
@@ -989,12 +989,9 @@
         {CLEAR_VARIABLE base_three}
         {CLEAR_VARIABLE base_two}
         {CLEAR_VARIABLE base_one}
-
-        [endlevel]
```

Comment:
@rodolphecombe is this intentional? The removal of the endlevel tag?

Reply:
Yes. [endlevel] tag makes no sense inside a victory event as it is called by an [endlevel] command